bin/requirements-checker.php

```diff
@@ -22,7 +22,14 @@
 
 $lineSize = 70;
 $isVerbose = in_array('-v', $argv) || in_array('-vv', $argv) || in_array('-vvv', $argv);
-$symfonyRequirements = new SymfonyRequirements(dirname(dirname(realpath($autoloader))));
+
+if (class_exists('\Symfony\Component\HttpKernel\Kernel') && defined('\Symfony\Component\HttpKernel\Kernel::VERSION')) {
```

Comment:
`VERSION` is always defined, so the check can be removed IMHO.

Reply:
So, the if condition could be replaced with:

```php
$symfonyVersion = class_exists('\Symfony\Component\HttpKernel\Kernel') ? \Symfony\Component\HttpKernel\Kernel::VERSION : null
```

Reply:
Right, I fixed it